test/test_optim.py

```diff
@@ -495,10 +495,23 @@ def test_multi_tensor_optimizers(self):
 
                     optimizer.step()
 
+                state.append(optimizer.state)
                 res.append(model.parameters())
 
-            for p1, p2 in zip(res[0], res[1]):
-                self.assertEqual(p1, p2, atol=5e-5, rtol=0)
+            st_state = state[0]
+            mt_state = state[1]
+            for st_p, mt_p in zip(res[0], res[1]):
+                self.assertEqual(st_p, mt_p, atol=5e-5, rtol=0)
```

Comment:
@albanD I'm not quite sure why the params are ok, perhaps max_exp_avg_sq's impact on the params is not significant enough to cause differences of more than 5e-5(?)